packages/terra-alert/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Fixed console error
```

Comment:
Lets update this changelog entry to be specific to the error that was fixed. Maybe something like: change children wrapper to be div instead of p tag given children accepts type node

Reply:
Sounds like a good description to me: https://github.com/gabeparra01/terra-core/commit/18802ce604470dd9a7af6bdb6fbc534641567b96